src/app/courses/credits-course/course-credits.component.ts

```diff
@@ -0,0 +1,25 @@
+import { Component, OnInit, OnDestroy } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+
+@Component({
+  templateUrl: './course-credits.component.html'
+})
+export class CourseCreditsComponent implements OnInit, OnDestroy {
+  id: string;
+  private sub: any;
+
+  constructor(private route: ActivatedRoute) {}
+
+  ngOnInit() {
+    this.sub = this.route.params.subscribe(params => {
+       this.id = params['id']; // (+) converts string 'id' to a number
+
```

Comment:
can remove unsused spaces. Could you please remove extra lines ?

Reply:
@nirojdyola Fixed. See https://github.com/ole-vi/planet/pull/248/commits/4733b887a21cd47982383fb9d2d78d0184515e97